examples/document.html

```diff
@@ -139,11 +139,18 @@ <h1 class="u-no-margin--bottom">{{ document.title }}</h1>
       <div class="l-docs__sticky-container">
         <aside class="p-table-of-contents">
           <div class="p-table-of-contents__section">
-            <h4 class="p-table-of-contents__header">On this page</h4>
+            <h4 class="p-table-of-contents__header">On this page!!</h4>
```

Comment:
`<h4 class="p-table-of-contents__header">On this page!!</h4>`

Is the '!!' here intentional?

Reply:
No, that should be removed. Thanks for the catch! I just pushed a commit to remove it :)